src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -115,32 +115,47 @@ export const OnImageComponents: Story = {
         this.
```

Comment:
Can this sentence be clearer? Maybe "The following images have aspect ratios applied and also use `Tooltip`s with `shouldWrapChildren`. This should not be done.

Reply:
Also, it is not clear to me how these examples are not rendering properly. The square, for example, looks square, etc...

Reply:
Admittedly, I don't recall the exact problem. I think it has to do with where the `aria-describedby` attribute is applied. That said, I am going to rewrite this whole thing because tooltip should not be applied directly to `Image` components. I updated this example to have a link around the image and now the `shouldWrapChildren` prop doesn't do much of anything.